kernel/Cargo.toml

```diff
@@ -82,7 +82,7 @@ which = "4"
 cc = "1.0"
 
 [features]
-default = ["addr2line", "serde", "serde_cbor"]
+default = ["addr2line", "serde", "serde_cbor", "core2", "abomonation", "rpc/default"]
```

Comment:
I was trying to avoid extra default dependencies for when we're not using the exokernel/controller so the normal nrk build wouldn't include things it doesn't strictly need. But maybe this is okay, anyways?

Reply:
Thank you for the reviews!

Now there are no features distinguishing between native build and exokernel build (just if/else on the code path). So, we need to include these features in the default.

Reply:
That makes sense!